### DISCUSSION TITLE
[Bug Report] 'make generate' fails importing

### DISCUSSION THREAD

**Discussion by capt-jack-aubrey** [stashapp/stash#4990] [Q&A]
Following the instructions from [Building from source](https://github.com/stashapp/stash/blob/develop/docs/DEVELOPMENT.md#commands), make generate fails as Yamashou/gqlgenc/client package is not found. 

`$ make generate`
[...]
`go: github.com/stashapp/stash/pkg/scraper/stashbox imports
        github.com/Yamashou/gqlgenc/client: module github.com/Yamashou/gqlgenc@latest found (v0.23.1), but does not contain package github.com/Yamashou/gqlgenc/client
tidy failed: go mod tidy failed: exit status 1
exit status 1
cmd/stash/main.go:1: running "go": exit status 1
make: *** [Makefile:282: generate-backend] Error 1`

**Fix**
It looks like there's a new version of the package named `clientv2`: https://pkg.go.dev/github.com/Yamashou/gqlgenc/clientv2
and the import statement simply needs to be updated. 

This is my first time working in Go otherwise i'd just submit a PR...will submit a fix later if i'm confident about it though.

**Comment by WithoutPants** [comment]
I tried to simulate this by running in a fresh docker container and I didn't get this error.

Go should be resolving `gqlgenc` to `github.com/Yamashou/gqlgenc v0.0.6` based on the entry in `go.mod`. I'm not quite sure what's going on, but it's looking more like a local environment thing.
